sites/fast-website/src/app/components/fast-frame/fast-frame.ts

```diff
@@ -161,12 +168,21 @@ export class FastFrame extends FASTElement {
         this.backgroundColor = this.previewBackgroundPalette[this.lastSelectedIndex];
     };
 
+    private resizeHandler = (): void => {
+        // Reset expanded when screensize is greater than 660
+        if (window.innerWidth > 660) {
```

Comment:
does this align to a breakpoint? If so - should this be using a stored value instead of a hard-coded number?

Reply:
yeah, I'll add this to this file since it's a unique breakpoint to this view.